<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 0);
font: 800 18pt &quot;Cantarell&quot;;</string>
      </property>
      <property name="text">
       <string>ESC - Sair</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_estornar">
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="txt_estorno">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTableWidget" name="tbl_estorno">
      <column>
       <property name="text">
        <string>Volume</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Produto</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_msg">
      <property name="styleSheet">
       <string notr="true">font: 800 22pt &quot;Cantarell&quot;;</string>
      </property>
      <property name="text">
       <string>Aguardando...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
